src/partials/contacts.html

```diff
@@ -0,0 +1,66 @@
+<section class="contacts">
+    <div class="contacts__container">
+    <h2 class="contacts__title">Contacts</h2>
+    <ul class="contacts__box">
+        <li class="contacts__card">
+            <p class="contacts__category contacts__category--cafe">CAFE</p>
```

Comment:
usuną bym tu contacts__category--cafe (ponieważ modifier pownien mówić co się dzieje z tym elementem to cafe za mało mówi) 
w scss
dodał bym contacts__category--cafe do contacts__category

Reply:
Nazwa klasy modyfikatora:

Musi być prosta, opisująca zmianę, która ma zostać wprowadzona.
Koniecznie jest tworzona zgodnie ze schematem blok--modyfikator lub blok__element--modyfikator.

<p class="alert alert--success"></p>
<p class="alert alert--error"></p>
<p class="alert alert--warning"></p>

Reply:
zrobione